Sources/UIKit/UIColor.swift

```diff
@@ -37,6 +37,27 @@ public extension UIColor {
         self.init(red: components.red, green: components.green, blue: components.blue, alpha: components.alpha)
     }
 
+    var hexString: String {
+
+        var components = UIColor.components(fromHex6: 0)
+        getRed(&components.red, green: &components.green, blue: &components.blue, alpha: &components.alpha)
+        let rgb: Int = (Int)(components.red*255)<<16
+            | (Int)(components.green*255)<<8
+            | (Int)(components.blue*255)<<0
+        return String(format:"#%06x", rgb)
+    }
+
+    var hexStringWithAlpha: String {
+
+        var components = UIColor.components(fromHex6: 0)
+        getRed(&components.red, green: &components.green, blue: &components.blue, alpha: &components.alpha)
+        let argb: Int = (Int)(components.alpha * 255)<<24
```

Comment:
Ok, so it seems that the `ARGB` format is the one that is more often expressed in 8 hexadecimal digits, following the pattern `#AARRGGBB`

The RGBA format can also be represented this way, but seems less frequent.

Source: https://en.wikipedia.org/wiki/RGBA_color_space